pairviewer.js

```diff
@@ -1,6 +1,7 @@
 // Apertium Global Pairviewer
 // Colin Pillsbury, Spring 2017
 // cpillsb1@swarthmore.edu
+// Modified from http://bl.ocks.org/tlfrd/df1f1f705c7940a6a7c0dca47041fec8
```

Comment:
This should be right above the versor.js portion. Sorry for the confusion.

Reply:
Ah okay